bin/publish-lts-rc.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+# TODO consider deleting in favor of binaries automatically deployed to https://repo.jenkins-ci.org/incrementals/org/jenkins-ci/main/jenkins-war/
```

Comment:
And presumably some other stuff in other repositories could be deleted, like `jenkins-stable-rc.mk` wherever that is.